@navikt/core/css/darkside/timeline.darkside.css

```diff
@@ -97,49 +103,113 @@
 }
 
 .navds-timeline__period--success {
-  background: var(--ac-timeline-period-success-bg, var(--a-surface-success-subtle));
-  box-shadow: inset 0 0 0 1px var(--ac-timeline-period-success-border, var(--a-border-success));
-}
+  background: var(--ax-bg-success-moderate);
+  border: solid 1px var(--ax-border-success);
+  color: var(--ax-text-success-icon);
+
+  &.navds-timeline__period--clickable:hover {
+    background: var(--ax-bg-success-moderate-hover);
+  }
+
+  &.navds-timeline__period--clickable:active {
+    background: var(--ax-bg-success-strong);
+    border: none;
+    color: var(--ax-text-success-contrast);
+  }
```

Comment:
For states like this where something gets selected on click (ToggleGroup as an example), we don't need `:active` since the "active" action is the element getting selected/pressed

Reply:
I guess they weren't in the design 🤔, I just felt it might help a tiny bit to show feedback (but I guess it's unnecessary since you're actively pressing... same rationale as not needing focus when clicking a button). :fire: removing!